common/components/BalanceSidebar/AddCustomTokenForm.jsx

```diff
@@ -34,7 +34,8 @@ export default class AddCustomTokenForm extends React.Component {
         </label>
         <input
           className={
-            'form-control input-sm ' + (this.state.symbol !== '' ? 'is-valid' : 'is-invalid')
+            'form-control input-sm ' +
```

Comment:
Most of the codebase has the smaller width prettier settings, so we will continue using that with Husky. 

Due to this, these formatting changes are fine to be included.